src/Microsoft.AspNetCore.Mvc.RazorPages/PageActionDescriptor.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Microsoft.AspNetCore.Mvc.RazorPages
 {
-    [DebuggerDisplay("{" + nameof(ViewEnginePath) + "}")]
+    [DebuggerDisplay("{ViewEnginePath = " + nameof(ViewEnginePath) + ", RelativePath = " + nameof(RelativePath) + "}")]
```

Comment:
You need to move the braces if you want to do this, they need to surround the `nameof()`s.

I'm personally super not a fan of nameof in these strings. it makes it an unreadable mess, and these names are public APIs and can't be changed anyway. You could also take the approach of moving the actually display to a method.

Reply:
Sure